src/utils.ts

```diff
@@ -195,6 +199,19 @@ export function readFileSync(filePath) {
   return fs.readFileSync(filePath, "utf-8");
 }
 
+export function doesPathExists(inputPath: string) {
+  return pathExists(inputPath);
+}
+export function readJson(jsonPath: string) {
+  return readJSON(jsonPath);
+}
+
+export function writeJson(jsonPath: string, object: any) {
+  return writeJSON(jsonPath, object, {
+    spaces: vscode.workspace.getConfiguration().get("editor.tabSize"),
```

Comment:
`get<T>(section: string): T | undefined;` this could return undefined, maybe we need to provide a default value here

Reply:
It should have a default value on vscode but added default here.